Documentation/user-guides/storage.md

```diff
@@ -175,14 +176,13 @@ metadata:
   name: example
 spec:
   replicas: 1
-  storage:
-    volumeClaimTemplate:
-      spec:
-        resources:
-          requests:
-            storage: 10Gi
+  paused: true
```

Comment:
What if we keep the existing manifest but adding `paused: true`?

```suggestion
  paused: true
  storage:
      volumeClaimTemplate:
      spec:
        resources:
          requests:
            storage: 10Gi
```

Reply:
I think it's good. However, it seems strange that it is set to 10 Gi in the first step. Shouldn't we use a smaller value  that 10Gi in the first step because we're going to increase it to 10 Gi in the next step? (because this is guide to increase pvc size)

So, how about specifying initial storage value less than 10Gi? (like 5Gi)
```suggestion
  paused: true
    storage:
      volumeClaimTemplate:
      spec:
        resources:
          requests:
            storage: 5Gi
```

Reply:
I might be wrong but I'm understanding correctly @simonpasquier is proposing in the first step adding the pause and updating the size of the volume all in 1 go. 

Then in the last step, we would just remove the pause and we could omit the `storage` section.

Reply:
> I might be wrong but I'm understanding correctly @simonpasquier is proposing in the first step adding the pause and updating the size of the volume all in 1 go.
Then in the last step, we would just remove the pause and we could omit the storage section.

I understand now. I think the procedure you suggested will work well also. I'll apply your feedback.

Reply:
What isn't very explicit in the current documentation is that we assume the initial value is less than 10Gi. Maybe using a patch command would be clearer?

```
kubectl patch prometheus/example --patch '{"spec": {"paused": true, "storage": {"volumeClaimTemplate": {"spec": {"resources": {"requests": {"storage":"10Gi"}}}}}}}' --type merge
```

Reply:
It's okay to change it to a patch command, but I think it became clear now without changing the yaml file to a patch command. 
  
What I misunderstood at first was that I wrote a guide to change the pvc size in the last step, but I didn't understand that your suggestion meant to change it in the first step.
  
Now, I think it's pretty clear because I modified it to change the pvc size in the first stage.
and mentioned like below at first step:
> And update the storage request in the `spec.storage` field of the custom

Reply:
I think it's now clear, but if you think it's better to change it to patch command, please let me know.